packages/devtools/devtools-extension/project.json

```diff
@@ -11,6 +11,30 @@
         "{options.outputPath}"
       ]
     },
+    "e2e": {
+      "executor": "@dxos/test:run",
+      "options": {
+        "buildTarget": "devtools-extension:bundle",
+        "coveragePath": "coverage/packages/devtools/devtools-extension",
+        "extensionPath": "packages/devtools/devtools-extension/out/devtools-extension",
+        "incognito": false,
```

Comment:
what happens if incognito isn't false?

Reply:
The extension does not have permission to run in incognito, and I did not find a way to add that permission